test/guess-number.test.js

```diff
@@ -0,0 +1,43 @@
+import compareNumbers from '../src/guess-number.js';
+
+const test = QUnit.test;
+
+test('compare-number', (assert) => {
+    // Arrange
+    const guess = 10;
+    const correctNumber = 32;
+    const expected = -1;
+
+    // Act
+    const result = compareNumbers(guess, correctNumber);
+
+    // Assert
+    assert.equal(result, expected);
+});
+
+test('compare-number', (assert) => {
+    // Arrange
+    const guess = 10;
```

Comment:
Looks like your first test and your second test are the same. I'm going to take a point off since you didn't test all cases for the compareNumber function, but if you change this test to test for the third case and resubmit, you can earn that point back! just slack me whenever you make the change.